.github/workflows/test.yml

```diff
@@ -28,11 +29,12 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v4
         with:
+          cache: false # setup-go v4 caches by default
           go-version: ${{ matrix.go-version }}
 
       - name: Run Test
         uses: nick-fields/retry@v2
         with:
           max_attempts: 3
           timeout_minutes: 15
-          command: go test ./... -v -race -count=1
+          command: go test -v -race ./...
```

Comment:
Why is `count=1` removed?

Reply:
This makes tests run concurrently again. Not only to make test-runs faster but also to spot data races.
See the relevant commit message: https://github.com/gofiber/fiber/pull/2469/commits/1e268a25039d671fb6fb84684985eedd26fa3a19

This was added in https://github.com/gofiber/fiber/commit/9effdf829a79df49c24318bf753f8332d6e75375 (the commit message doesn't match what has actually changed, maybe @ReneWerner87 can explain why this is/was needed)

Reply:
I also removed the `test` job retry logic in 6de93415ef9030f930118085e705ba2532c7d12c which allows us to spot data races immediately.

Reply:
btw, this change just detected a data race! 🥳 4c4d3475a82fb618902d2594c5f12d0c8fede7d2

Reply:
this prefix was used to disable the golang cache for testing, because in the past several tests went through it

for now, would get this construct

Reply:
> for now, would get this construct

with "-count=1"